vega_sim/environment/agent.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         self,
         wallet_name: str,
         wallet_pass: str,
+        key_name: Optional[str] = None,
```

Comment:
Can we add a short bit to the docstring too?

Reply:
Have added:

Reply:
I think this seems reasonable. Letting people know the 'ultimate' default rather than necessarily what is immediately used seems fine to me